### ISSUE TITLE
Is it open source?

### ISSUE THREAD

**Issue by WMnologo** [MickiusMousius/Rolio46Keyboard#16]
![image](https://github.com/user-attachments/assets/8b64c56f-bf16-4a2b-8c2c-25347e866f47)
Rolio46 is so cool, is this CNC file open source?

**Comment by WMnologo** [comment]
![image](https://github.com/user-attachments/assets/8e329181-e2f7-4c76-b025-ecba66285105)
I searched and couldn't find one of the files. Is it open source

**Comment by MickiusMousius** [comment]
Ah, that one is not yet open source yet. But I’ll publish it soon. I have one more iteration to make it easier to assemble and improve Bluetooth performance.

I do have some inventory on hand though.

**Comment by WMnologo** [comment]
﻿
Okay, looking forward to your open source. It's really cool and I really like it.

**Comment by WMnologo** [comment]
![image](https://github.com/user-attachments/assets/e65f0957-9df8-4aed-9bef-0f03060bd449)
Can you share this  model? I want the keyboard to be easy to carry around

**Comment by MickiusMousius** [comment]
I shall, if you’re building one now let me know, otherwise I’ll try to batch it together with a bunch of other updates.

**Comment by WMnologo** [comment]
Yes, I made one. Now I want this shell to be easy to carry around

**Comment by MickiusMousius** [comment]
Did you make a CNC chassis or a 3D printed one?

the two travel cases have different internal dimensions.

**Comment by WMnologo** [comment]
3D printed

**Comment by WMnologo** [comment]
hello

**Comment by MickiusMousius** [comment]
Hey, I think I have the file. But can not attach a step file here it seems.

I don’t want to put it in the repo as I have a heavily revised FDM chassis (no glue needed now) and it requires a different case.

How would you like for me to share the file to you?

**Comment by WMnologo** [comment]
Have his 3D files and PCB files been open source? I try one. It's so cool.
![image](https://github.com/user-attachments/assets/a51c533d-f492-483c-909d-d6c5bbebff1e)
